conf/messages

```diff
@@ -27,7 +27,7 @@ landingPage.howTo.list1.bullet1=report the disposal of UK residential property o
 landingPage.howTo.list1.bullet2=pay any tax you might owe for that disposal
 landingPage.howTo.list1.bullet3=view your previous and current returns
 
-landingPage.howTo.firstReturnsOnly=At the moment, only the first return for UK property sales or disposals in the tax year can be submitted online. If you need to amend an existing return or complete any further returns for the tax year you will need to <a href="{0}">contact HMRC</a>.
+landingPage.howTo.firstReturnsOnly=If you need to amend an existing return you will need to <a href="{0}">contact HMRC</a>.
```

Comment:
This content should be feature flagged - we don't want it to show if further returns are not enabled. There's already a `furtherReturnsEnabled` flag in ` ViewConfig` which you can use.